.github/workflows/test.yaml

```diff
@@ -7,10 +7,11 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macOS-latest]
+        cc: [clang, gcc]
```

Comment:
Are you sure there will be a `gcc` installed on `macOS-latest`, or a `clang` installed on `ubuntu-latest`?

Reply:
Yep!